scripts/k8s_persistent_dir_cleaner.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: persistent-dir-cleaner
+  labels:
+    app: nginx
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:latest
```

Comment:
why nginx?

Reply:
it's for internal use so it doesn't really matter but what do you suggest?